app/views/dashboards/_actions.html.erb

```diff
@@ -4,6 +4,11 @@
     <span class="crayons-indicator"><%= @user.articles_count %></span>
   </a>
 
+  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "series" && (params[:which] == "organization" || params[:which].blank?) %>" href="/<%=current_user.username%>/series">
```

Comment:
**suggestion (non-blocking):** This line is quited long and hard to read. Maybe something like this instead?

```suggestion
  is_own_series = params[:action] == "series" && (params[:which] == "organization" || params[:which].blank?)
  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if is_own_series %>" href="/<%=current_user.username%>/series">
```